Source/WebKit/UIProcess/ios/WKContentView.mm

```diff
@@ -264,6 +273,15 @@ - (instancetype)_commonInitializationWithProcessPool:(WebKit::WebProcessPool&)pr
 #if HAVE(VISIBILITY_PROPAGATION_VIEW)
 - (void)_setupVisibilityPropagationViewForWebProcess
 {
+#if USE(EXTENSIONKIT)
+    if (!_visibilityPropagationInteractionForWebProcess) {
+        SEL selector = NSSelectorFromString(@"createVisibilityPropagationInteraction");
+        if ([_page->process().extensionProcess().get() respondsToSelector:selector])
```

Comment:
Will there be a time when we can switch to the compile-time selector check? Maybe we should file a bug to remember to clean this up, since this is a little more costly than the @selector syntax (or just calling the method if we know the SDK supports it).

Reply:
Yes, that is a good point. I will file a bug for this.

Thanks for reviewing!

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=266069 for this.